### ISSUE TITLE
[BUG] Downloads stop displaying status, and Error

### ISSUE THREAD

**Issue by bewbylover** [Jules-WinnfieldX/CyberDropDownloader#738]
**Describe the bug**
Crashed, and before the crash downloads stopped showing progress they showed up as pictured.  Here is the error message in text:

```
An error occurred, please report this to the developer
unhandled errors in a TaskGroup (1 sub-exception)
  + Exception Group Traceback (most recent call last):
  |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/main.py", line 85, in director
  |     await runtime(manager)
  |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/main.py", line 43, in runtime
  |     async with asyncio.TaskGroup() as task_group:
  |   File "/usr/lib/python3.11/asyncio/taskgroups.py", line 147, in __aexit__
  |     raise me from None
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/utils/utilities.py", line 56, in wrapper
    |     return await func(self, *args, **kwargs)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/scraper/crawler.py", line 109, in forum_login
    |     raise FailedLoginFailure(status=403, message="Failed to login after 5 attempts")
    | cyberdrop_dl.clients.errors.FailedLoginFailure: 403
    | 
    | During handling of the above exception, another exception occurred:
    | 
    | Traceback (most recent call last):
    |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/scraper/crawler.py", line 52, in run
    |     await self.fetch(item)
    |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/scraper/crawlers/socialmediagirls_crawler.py", line 63, in fetch
    |     await self.forum_login(login_url, session_cookie, username, password, wait_time)
    |   File "/home/kris/Downloads/Cyberdrop_DL.V5.1/venv/lib/python3.11/site-packages/cyberdrop_dl/utils/utilities.py", line 62, in wrapper
    |     await log(f"Scrape Failed: {args[0].url} (Failed Login)", 40)
    |                                 ^^^^^^^^^^^
    | AttributeError: 'URL' object has no attribute 'url'
    +------------------------------------
```

**To Help Reproduce**
Steps to help reproduce the behavior:
1. Start program with a number of simp, nudo, xbunker, and social forum links.
2. Just wait.

**Screenshots**
![Screenshot from 2023-12-18 15-20-12](https://github.com/Jules-WinnfieldX/CyberDropDownloader/assets/152096820/843fd66e-ab42-494e-914e-a7743ee52c77)

**Setup Information**
 - OS: Ubuntu 20.04.6 LTS
 - python 3.11.6
 - Cyberdrop Downloader V5.1.8

**Downloader.log file**
[downloader.log](https://github.com/Jules-WinnfieldX/CyberDropDownloader/files/13709367/downloader.log)

**Additional context**

**Comment by Jules-WinnfieldX** [comment]
Should be fixed in 5.1.9 going up now

**Comment by bewbylover** [comment]
Crashed before the scrape now
![Screenshot from 2023-12-18 16-31-56](https://github.com/Jules-WinnfieldX/CyberDropDownloader/assets/152096820/1396bff8-8936-4be6-9fa3-0d59f8fecb13)

[downloader.log](https://github.com/Jules-WinnfieldX/CyberDropDownloader/files/13709700/downloader.log)
